steps.md

```diff
@@ -1,8 +1,197 @@
-- Turn on Strict Options
-  - ![](https://cl.ly/1e32a57f132d)
-- Identify needed types 
-  - Obvious (red squiggles)
-    - ![](https://cl.ly/92c33a4dc8fe)
-  - Non-obvious (lack of constraints)
-    - 
-- Annotate state
\ No newline at end of file
+# Redux from scratch with Typescript
+
+The aim of this is to walk through adding types to a basic Redux example and then improve on those types as a means to explain a number of basic and advanced concepts in Typescript.
+
+---
+
+## 0 - Humble Beginnings
+
+This section shows the starting point of the Redux example a plain Javascript file.
+
+### New Concepts
+
+_None_
+
+### Observations
+
+- Basic implementation of Redux
+- No types
+
+---
+
+## 1 - Enter Typescript
+
+This section changes the Javascript file to Typescript and talks about the errors that now appear.
+
+### New Concepts
+
+- compiler errors
+- any
+- strict mode
+
+### Observations
+
+- Change file to `.ts`
+- No types yet
+- Some errors show up
+  - 3 implicit any errors (`item`, `id`, and `action`)
```

Comment:
Do you think we should include `line:column` numbers?

Reply:
@roblafeve Thats a good idea, I can work on that.

Reply:
I tried it out, might not be worth it?